2020-03-Autumn/3_backtracking/README.md

```diff
@@ -58,21 +58,37 @@ Go to [Solution](#s1)   [Top](#top)
 <!-- Don't remove -->
 <a name="p2"/>
 
-### 2. PROBLEM 2 TODO :bug:
+### 2. PROBLEM 2 Subsets
 
-Source: TODO :bug:
+Source: [LeetCode](https://leetcode.com/problems/subsets/)
 
 #### Scenario
 
-Problem Statement TODO :bug:
+Given an integer array nums, return all possible subsets (the power set).
```

Comment:
you're missing important an important constraint: 
> The solution set must not contain duplicate subsets. Return the solution in any order.

if you decide to change the problem statement, pls change your solution accordingly.